test/integration/protocol_integration_test.cc

```diff
@@ -1117,6 +1117,9 @@ TEST_P(DownstreamProtocolIntegrationTest, InvalidContentLengthAllowed) {
         hcm.mutable_http2_protocol_options()
             ->mutable_override_stream_error_on_invalid_http_message()
             ->set_value(true);
+        hcm.mutable_http_protocol_options()
+            ->mutable_override_stream_error_on_invalid_http_message()
+            ->set_value(true);
```

Comment:
@alyssawilk Should the expectation here be that if `override_stream_error` is set to `true`, then the HTTP/1.1 connection is left open? If I understand correctly, the connection will still be closed because the HCM is set to a draining before the request is complete.

Reply:
yeah, we may have to send a different type of invalid message, one which will be rejected by the HCM (where the headers will be complete). For example, fail to upgrade?  If you snag the upgrade headers from the websocket test and include content length 0 that may work

Reply:
hm, isn't that the case already for the integration tests I added, where the message is missing the `host` header?